### PR TITLE
Added language urls to Play, F-Droid and GitHub

### PR DESCRIPTION
**Pull request by PanderMusubi**
As requested full overview of languages with urls

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,23 +69,140 @@ <h1 class="header">AnySoftKeyboard</h1>
...
-<li>Swiss</li>
-<li>Ukrainian</li>
-</ul>
+<table>
+<tr><td>English (QWERTY, DVORAK, AZERTY and Colemak)</td>
+<td>built-in</td>
+<td>built-in</td>
+<td><a href="https://github.com/AnySoftKeyboard/AnySoftKeyboard"><img src="images/github.png" /></a></td></tr>
+<tr><td>Hacker's Keyboard (SSH)</td>
+<td><a href="https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard"><img src="images/play.png" /></a></td>
```
Comment (menny) [inline index.html]: Why is there a link to Hacker's keyboard here? It a full keyboard app, bot a pack (as far as I know)

**Comment by PanderMusubi** [comment]
Ah, yes but the app does nothing, you need SSH app VX ConnectApp with it. That is actually the one that pulls in the terminal app. But the terminal app has no launcher and the keyboard is available in AnySoftKeyboard.

**Comment by PanderMusubi** [comment]
Can you live with that for now? This table for sure will get plenty of updates. After merging, please share it with the language pack community so they can contribute improvements.

**Comment by PanderMusubi** [comment]
Hope this is better for a merge. Let me know.

**Pull request merged**